metric_learn/itml.py

```diff
@@ -167,6 +181,17 @@ class ITML_Supervised(_BaseITML, TransformerMixin):
 
   Attributes
   ----------
+  bounds_ : `list` of two numbers
+      Bounds on similarity, aside slack variables, s.t. ``d(a, b) < pos`` for
+      all given pairs of similar points ``a`` and ``b``, and
+      ``d(c, d) > neg`` for all given pairs of dissimilar points ``c`` and
+      ``d``, with ``bounds=[pos, neg]``, and ``d`` the learned distance. If
+      not provided at initialization, these are the ones derived at train
+      time.
+
+  n_iter_ : `int`
+      The number of iterations the solver has ran.
```

Comment:
"has ran" -> "has run"

Reply:
>As currently implemented, bounds_ should be an ndarray-like of shape (2,).
>
>We should probably update the code to accept list or tuple types for the bounds as well.

That's right, I forgot about that. For now I'll just change the docstring, but I'll open an issue to get this thing straight

> "these are the ones" -> "these are"

Thanks, will do 


> "has ran" -> "has run"

That's right, thanks